### PR TITLE
Feature/my profile route

### PR DESCRIPTION
**Pull request by MTCawthray**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- This branch adds functionality to the buttons in the NavBar component.
- Now, when a user clicks on the ```My Profile``` button from any page view, they are taken to their profile page.
- I also added logout functionality to the ```log out``` button when a user is in the friends view

## What (if anything) did you refactor?
- NA

## Were there any issues that arose?
- NA

## Is there anything that you need from your teammates?
- NA

## Any other comments, questions, or concerns?
- NA

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [ ] dev tools
- [ ] Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import './Error.css';
```
Comment (hayleyw7) [inline src/components/Error/Error.js:1]: Thanks for starting this!

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ import React from 'react';
...
-const NavBar = ({logOut}) => {
+const NavBar = ({logOut, id}) => {
 
   return (
     <ul className="nav-bar">
       <li>
 
         <Link
-          to={'/profile'}
+          to={`/profile/${id}`}
```
Comment (nataliemcintyre2021) [inline src/components/NavBar/NavBar.js:12]: Nice updating this link!

File: src/components/Loader/Loader.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import loading from '../../assets/loading.gif'
+import loading from '../../Assets/loading.gif'
```
Comment (hayleyw7) [inline src/components/Loader/Loader.js:2]: This is driving me nuts, but I know we are all renaming it right now haha

File: src/components/UserDisplay/UserDisplay.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import {useState, useEffect} from 'react';
 import {useQuery} from '@apollo/client';
 import {GET_ALL_USERS} from '../../graphQL/queries';
 
-const UserDisplay = ({ friends }) => {
+const UserDisplay = ({ friends, logOut, id }) => {
```
Comment (nataliemcintyre2021) [inline src/components/UserDisplay/UserDisplay.js:9]: Awesome getting this working!

**Review by nataliemcintyre2021 — APPROVED** [review]

**Review by hayleyw7 — APPROVED** [review]
Thanks for much for your quick, efficient, and thoughtful work on this.

**Pull request merged**